entry/src/main/js/default/pages/index/index.js

```diff
@@ -32,10 +38,57 @@ export default {
         this.notifications = !this.notifications;
         console.log("Notifications toggled !");
     },
+    fetchDateAndTime : function(){
+        const date = new Date();
+        this.date_d=(String(date.getDate()).padStart(2, '0'))
+        this.date_m=(String(date.getMonth()+1).padStart(2, '0'))
+        this.time_h=(String(date.getHours()).padStart(2, '0'))
+        this.time_m=(String(date.getMinutes()).padStart(2, '0'))
+        this.time_h_0 = this.time_h.toString().substring(0, 1);
+        this.time_h_1 = this.time_h.toString().substring(1, 2);
+        this.time_m_0 = this.time_m.toString().substring(0, 1);
+        this.time_m_1 = this.time_m.toString().substring(1, 2);
+        const dayOfWeek = (date.getDay())
+        if (dayOfWeek==1)
+        this.date_w="Mon"
+        else if (dayOfWeek==2)
+        this.date_w="Tue"
+        else if (dayOfWeek==3)
+        this.date_w="Wed"
+        else if (dayOfWeek==4)
+        this.date_w="Thu"
+        else if (dayOfWeek==5)
+        this.date_w="Fri"
+        else if (dayOfWeek==6)
+        this.date_w="Sat"
+        else
+        this.date_w="Sun"
+    },
+    fetchData : function(){
+        let data;
+        fetch.fetch({
+            url:'https://neumorphism-api.herokuapp.com/neumorphism/smart_watch',
```

Comment:
Try to get weather data from openweathermap.org and display it

Can refer to,
https://forums.developer.huawei.com/forumPortal/en/topic/0201541977865920174